gsk4/src/auto/renderer.rs

```diff
@@ -95,30 +84,6 @@ impl<O: IsA<Renderer>> RendererExt for O {
         }
     }
 
-    fn render<P: IsA<RenderNode>>(&self, root: &P, region: Option<&cairo::Region>) {
```

Comment:
These should probably not all disappear, otherwise the whole of GSK becomes useless.

Can you add a manual implementation for them into `src/renderer.rs` into a `RendererExtManual` trait?

Reply:
Ah good catch. I assumed it was already the case from the `Gir.toml` as it mentions `RendererManualExt` (which is wrong btw)